server/cluster/metadata/types.go

```diff
@@ -74,6 +74,10 @@ type DropTableResult struct {
 	ShardVersionUpdate []ShardVersionUpdate
 }
 
+type DropTableMetadataResult struct {
+	Table storage.Table
```

Comment:
When to use this field?

Reply:
It is used to tell the user the information of the table being dropped when the partition table is dropped.